plugins/UM3NetworkPrinting/NetworkPrinterOutputDevice.py

```diff
@@ -1133,6 +1139,27 @@ def _onFinished(self, reply):
                 else:
                     pass
 
+            elif self._api_prefix + "system" in reply_url:
+                # Check if the printer has time, and if this has a valid system time.
+                try:
+                    data = json.loads(bytes(reply.readAll()).decode("utf-8"))
+                except json.decoder.JSONDecodeError:
+                    Logger.log("w", "Received an invalid authentication request reply from printer: Not valid JSON.")
+                    return
+                if "time" in data and "utc" in data["time"]:
+                    try:
+                        printer_time = gmtime(float(data["time"]["utc"]))
+                        Logger.log("i", "Printer has system time of: %s", str(printer_time))
+                    except ValueError:
+                        printer_time = None
+                    if printer_time is not None and printer_time.tm_year < 1990:
```

Comment:
If printer_time is none, it's also likely to be not valid right?

Reply:
Or the API returned something we don't understand. Figured I opt for the safe route of not handling "bad" data in any way other then doing nothing with it.

Reply:
Fair enough.